src/Draco.Compiler/Internal/Solver/MemberConstraint.cs

```diff
@@ -57,12 +64,21 @@ public override IEnumerable<SolveState> Solve(DiagnosticBag diagnostics)
                 .WithFormatArgs(this.MemberName, this.Unwrap(this.Accessed));
             // We still provide a single error symbol
             var errorSymbol = new UndefinedMemberSymbol();
+            this.Unify(this.MemberType, new ErrorTypeSymbol("<error>"));
```

Comment:
This isn't necessarily correct, overloading is a thing for members. It likely gets ignored tho, so if it's beneficial for other cases then it can stay.